p360monitoring.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+if (!defined('WHMCS')) {
+    die('This file cannot be accessed directly');
+}
+
+require __DIR__ . '/vendor/autoload.php';
+
+use WHMCS\Module\Server\Plesk360Monitoring\KaApi;
+use WHMCS\Module\Server\Plesk360Monitoring\Logger;
+use WHMCS\Module\Server\Plesk360Monitoring\Params;
+use WHMCS\Module\Server\Plesk360Monitoring\Plans;
+use WHMCS\Module\Server\Plesk360Monitoring\Plans\ProPlan;
+use WHMCS\Module\Server\Plesk360Monitoring\Translator;
+
+Translator::loadTranslations();
+
+function p360monitoring_MetaData()
```

Comment:
Are we allowed to add return types to this functions?

Reply:
No access to the source code, the docs are without return types, but I checked - hook functions works with added types